### DISCUSSION TITLE
征集关于下个宣传视频的建议

### DISCUSSION THREAD

**Discussion by HSLix** [HSLix/LixAssistantLimbusCompany#24] [General]
rt，上个视频虽然个人觉得还行，但就是过不了阿b的申鹤（其它地方都过了）

我想，宣传视频要是像大部分视频那种，长、比较无聊的形式就很没意思，所以借了爱丽丝速通来制作

现在没什么灵感，欢迎看到这个讨论的同学提出自己喜欢看的、宣传视频的形式，最好有案例啦
（不知下次有时间和心思搞视频是什么时候啦）

**Comment by linzeduan** [comment]
我觉得不要被别人拿去商用就挺好了，过度宣传其实也不太好，毕竟是脚本，随缘吧。只把他当成自己的一个小练手

**Comment by HSLix** [comment]
> 我觉得不要被别人拿去商用就挺好了，过度宣传其实也不太好，毕竟是脚本，随缘吧。只把他当成自己的一个小练手

你说得有理，当时担心做出来没人用，也就没有反馈来改进，现在情况不同了。

不过商用这点，作为开源项目难免了，而且该项目用的GPL没禁止商用，我只希望代码开源，促进开源生态发展。

不过都商用了，人家怎会愿意开源呢，唉

**Comment by ShuoYang-buct** [comment]
好欸。但是用不了力 悲

**Reply by HSLix** [reply]
用不了吗？2.1.10还挺流畅的吧，托下巴.jpg

**Reply by ShuoYang-buct** [reply]
用不了哇，它停在镜牢的选择页面不进去，就是左边是镜3普通，右边是镜3困难，中间是选人升级光的那个页面不动

**Reply by HSLix** [reply]
> 用不了哇，它停在镜牢的选择页面不进去，就是左边是镜3普通，右边是镜3困难，中间是选人升级光的那个页面不动

有问题请按要求提issue
